Taarafo.Core/Brokers/Storages/IStorageBroker.Groups.cs

```diff
@@ -6,14 +6,19 @@
 using System;
 using System.Linq;
 using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Taarafo.Core.Models.GroupMemberships;
 using Taarafo.Core.Models.Groups;
 
 namespace Taarafo.Core.Brokers.Storages
 {
-    public partial interface IStorageBroker
-    {
+	public partial interface IStorageBroker
+	{
+        DbSet<GroupMembership> GroupMemberships { get; set; }
+
         ValueTask<Group> InsertGroupAsync(Group group);
+        void onModelCreating(ModelBuilder modelBuilder);
         IQueryable<Group> SelectAllGroups();
-        ValueTask<Group> SelectGroupByIdAsync(Guid groupId);
-    }
+		ValueTask<Group> SelectGroupByIdAsync(Guid groupId);
+	}
```

Comment:
We shouldn't change anything in here because we are only working with `GroupMemberships`

Reply:
@KFitz114 This is still pending